kuma/wiki/views.py

```diff
@@ -2186,7 +2187,11 @@ def _make_first_edit_email(new_rev, request):
     template = 'wiki/email/edited.ltxt'
     context = context_dict(new_rev)
     message = render_to_string(request, template, context)
-    email = EmailMessage(subject, message, settings.DEFAULT_FROM_EMAIL)
+    article_url = "%s%s" % (Site.objects.get_current().domain,
+                            doc.get_absolute_url())
+    email = EmailMessage(subject, message, settings.DEFAULT_FROM_EMAIL,
+                         headers = {'X-Article-Url': article_url,
+                                    'X-Editor-Username': user.username})
```

Comment:
While RFC 822 is not mentioning it, it's good practice to let the extension headers contain the product they are from, so let's use `X-Kuma-Document-URL` and `X-Kuma-Document-Editor`.